changelog/27927.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:improvement
+storage/s3: Pass context to AWS SDK calls
+storage/dynamodb: Pass context to AWS SDK calls
+```
```

Comment:
```suggestion
```release-note:improvement
storage/dynamodb: Pass context to AWS SDK calls
```

```release-note:improvement
storage/dynamodb: Pass context to AWS SDK calls
```
```